tests/unit/test_work_item_properties.py

```
@@ -644,3 +644,127 @@ def test_create_file_property(
+                created.id,
+                UpdateWorkItemProperty(description="Updated project property"),
+            )
+            assert updated.id == created.id
+            assert updated.description == "Updated project property"
+        finally:
+            try:
+                client.work_item_properties.delete_project(
+                    workspace_slug, project.id, created.id
+                )
+            except Exception:
+                pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)